### PR TITLE
Update and fix instructions to build the docs

### PR DESCRIPTION
**Pull request by santisoler**
#### Summary

Update the instructions on how to build the documentation pages and serve them locally.

#### PR Checklist
* [ ] If this is a work in progress PR, set as a Draft PR
* [ ] Linted my code according to the [style guides](https://docs.simpeg.xyz/latest/content/getting_started/contributing/code-style.html).
* [ ] Added [tests](https://docs.simpeg.xyz/latest/content/getting_started/contributing/testing.html) to verify changes to the code.
* [ ] Added necessary documentation to any new functions/classes following the
      expect [style](https://docs.simpeg.xyz/latest/content/getting_started/contributing/documentation.html).
* [ ] Marked as ready for review (if this is was a draft PR), and converted
      to a Pull Request
* [ ] Tagged ``@simpeg/simpeg-developers`` when ready for review.

### TODO

- [ ] Fix the ref link

### CODE REVIEW

File: docs/content/getting_started/contributing/documentation.rst [hunk 1]
```diff
@@ -70,18 +70,44 @@ For example:
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you would like to see the documentation changes.
-In the repo's root directory, enter the following in your terminal.
+You can build the documentation pages locally to check how new changes would
+look like. First, make sure that you have :ref:`created and activated an
```
Comment (jcapriot) [inline docs/content/getting_started/contributing/documentation.rst:74]:
```suggestion
You can build the documentation pages locally to see how the new changes will look. 
First, make sure that you have :ref:`created and activated an
```

Comment (jcapriot) [reply docs/content/getting_started/contributing/documentation.rst:74]: The original statement just sounded odd to me...

Also your `:ref:` link is broken

**Review by jcapriot — CHANGES_REQUESTED** [review]

**Comment by santisoler** [comment]
@jcapriot, I see the `:ref:` link working properly. Is it broken on your side?

**Comment by jcapriot** [comment]
Yeah, it looks good. I think I was initially caught off guard by the length of the link text,

**Review by jcapriot — APPROVED** [review]

**Pull request merged**